metis_client/namespaces/v0_collections.py

```diff
@@ -40,6 +40,7 @@ async def create_event(
             description=opts.get("description", ""),
             dataSources=opts.get("data_source_ids", []),
             users=opts.get("user_ids", []),
+            visibility=opts.get("visibility", "private"),
```

Comment:
This obviously should be ENUM.

Reply:
Nope. It's a string with type hint of allowed values.

Reply:
I think, you need to add
```
    visibility: NotRequired[MetisCollectionVisibility]
```
to the `MetisCollectionCreateDTO` and to the `MetisCollectionsCreateKwargs`.

Then type hint doesn't allow you to pass anything not allowed.

Reply:
Update tests:
```patch
diff --git a/tests/namespaces/test_v0_collections.py b/tests/namespaces/test_v0_collections.py
index 1ac2974..e06cbd4 100644
--- a/tests/namespaces/test_v0_collections.py
+++ b/tests/namespaces/test_v0_collections.py
@@ -41,6 +41,7 @@ PATH_C_PUT_PAYLOAD: MetisCollectionCreateDTO = {
     "dataSources": [1, 2, 3],
     "users": [1, 2, 3],
     "description": random_word(10),
+    "visibility": "private",
 }
 PATH_C_PUT_RESPONSE_PAYLOAD: MetisCollectionDTO = {
     **PATH_C_PUT_PAYLOAD,
@@ -203,6 +204,7 @@ async def test_create_collection(
         "description": PATH_C_PUT_PAYLOAD["description"],
         "data_source_ids": PATH_C_PUT_PAYLOAD["dataSources"],
         "user_ids": PATH_C_PUT_PAYLOAD["users"],
+        "visibility": PATH_C_PUT_PAYLOAD["visibility"],
     }
     with raises:
         col = await client_async.v0.collections.create(**kwargs)
```